build/gradle/commitAndProductionStage.gradle

```diff
@@ -390,6 +391,10 @@ def includeAdditionalFiles(artifacts, relsrcpath, productOutputPath, reldstpath,
 		    if (!Helper.buildForMac(project)) {
 		        ant.untar(src: artifactZipPath, dest: artifactInstallPath, compression: 'gzip')
 		    } else {
+				//It is certain that this code will run when creating build for mac.
+				//Therefore, the jre must be copied into the `Eclipse` folder of the bundled application.
+				artifactInstallPath = new File(productOutputPath.absolutePath ,"/Eclipse.app/Contents/Eclipse/${reldstpath}")
+				
 				// Use external tar command for OSX build since the jdk to bundle with HALE contains symbolic links
 				// This does only work on UNIX-based systems with tar installed!
 				def command = ['tar', '--preserve-permissions','-xzf', artifactZipPath.absolutePath]
```

Comment:
Here using tr is not a problem on Mac?

Reply:
Here using `tar` is no problem. In Mac `tar` gives problem only when used with `--transform` option.